docker/images/infra-task/Dockerfile

```
@@ -0,0 +1,15 @@
+ARG base_image=ruby:2.7.3
```

Can we use the latest patch version of the base image? (I'm assuming we have control of this because we're just running our code here as opposed to one of the GOV.UK apps that's stuck on an old patch release?)

Also worth trying the `-slim` version as it saves quite a lot on the image size.
```suggestion
ARG base_image=ruby:2.7.5-slim
```